internal/server/debug.go

```diff
@@ -6,14 +6,14 @@ import (
 
 	"github.com/gin-gonic/gin"
 
+	"github.com/infrahq/infra/api"
 	"github.com/infrahq/infra/internal/access"
 	"github.com/infrahq/infra/internal/server/models"
 )
 
-func pprofHandler(c *gin.Context) {
+func pprofHandler(c *gin.Context, _ *api.EmptyRequest) (*api.EmptyResponse, error) {
```

Comment:
Now that the database transaction comes from `wrapRoute` this handler had to be updated to fit the interface required by `wrapRoute`.